README.md

```diff
@@ -3,6 +3,34 @@
 [Homebridge Net Nanny](https://github.com/pricemi115/homebridge-grumptech-netnanny), by [GrumpTech](https://github.com/pricemi115/), is a [Homebridge](https://homebridge.io) dynamic platform plug-in that publishes metrics measuring the health of a network.
 This plugin was inspired by [Dave Hamilton's](https://twitter.com/DaveHamilton), of the [Mac Geek Gab](https://www.macgeekgab.com), 3-Ping strategy for evaluating network health.
 
+## What Problem Does Net Nanny Solve?
+
+When we suffer network problems, it's sometimes hard to figure out whether the problem is on the local area network, a problem with your Internet service provider, or a problem with a service to which you're trying to connect. In order to isolate the problem, a good practice is to open a terminal window and ping your router, then your modem, and then a known-reliable server on the Internet.
+
+The Net Nanny Homebridge plugin is designed to give you this information at a glance from within HomeKit. The Homebridge plugin creates a fake switch called Net Nanny, along with fake carbon dioxide (C02) sensors that will report your network health.
```

Comment:
C02 should be CO2....O for Oxygen

Reply:
Typo fixed

Reply:
Just an FYI, some (I among them) like to have a commit history that is as clean as possible. Since it would have been easy to amend commit # 494536c08aec8c1d7a114ecc26606993242e5a5e for this, that would have been the preference. Keeping a more "noisy" commit history on development branches is perfectly fine and recommended. Sometimes however, with more complex multi-commit pull requests, it is not always easy to keep all the changes resulting from the review associated with their original commit. As such, having a commit to address the review comments is acceptable.